tasks/Dictator Game.md

```diff
@@ -0,0 +1,61 @@
+
+Participants (or receivers) can be paid a flat rate, or can be paid for doing extra tasks.
+
+Dana et. Al. Variation:
+Private Condition: In this variation, the dictator is told that the amount they decide to allocate for their receiver is just given to them with a note attached stating, “Thank you for your time today. An additional $__ is attached to this note.” The exit option is still available.
+
+
+John A’s experiment included allocating all participants a baseline of $5, splitting all participants into two groups, A and B(overall split 50-50, i.e. each dictator has one participant, the same as the procedure discussed), and separating these groups into two rooms(our procedure discussed also allows no contact between the groups). The dictators were then given and addition $5 that they could allocate in $0.50 increments (notably this is still increments of 10%, the same as the procedure discussed).  
+
+Specific Variations from the procedure discussed:
+In one experiment, dictators could decide to take up to $1 from their respective participant, in another version of his experiment he allowed for dictators to take up to their participants entire allotted amount.  Subjects in group(room) B were told that participants in group A earned $10 for completing a task, but all participants earned money.
+
+# Procedure
+## Steps
+1. Participants are broken into groups A(Dictators) and B(Participants), and are separated, not allowing contact between the two groups, with the overall distribution being 50-50(i.e. each dictator has one receiver).
+2. Participants in group A (The dictators) are distributed written instructions about the game, they and their anonymous receiver will play(the instructions refer to both roles as “participants”).  Meanwhile group B is given unrelated tasks to complete.
+3. The Dictators are asked to distribute $10 of real money between them and their receiver, in $1 increments(i.e. 10-0, 9-1, 8-2, etc.).  (NOTE: the dictators instructions state that once a decision is made, the experimenter will distribute the same set of rules to the receiver, even if no money is allocated. Thus the receiver has common knowledge.)
+4. Once the dictator has made their choice they are informed of the “Exit Option”, where they take $9 and no further instructions are given to the receiver.
+5. The dictator makes a decision and is then dismissed.
+6. Regardless of the decision all parties are paid an allotted amount, and the game is over.
+
+
+## Roles 
+Dictator, given the moral decision to pay their receiver out of their own pocket.
+
+Receiver, given tasks to perform in idle time
+
+## Instructions
+Groups A and B (Dictators and Receivers) are not to interact, only interaction is between participant and experimenter.
```

Comment:
In this experiment each of you will be paired with a different person who is in another room. You will not be told who you are matched with during or after the experiment, and he or she will not be told who you are either during or after the experiment.

You will notice that there are other people in the same room with you who are also participating in the experiment. You will not be paired with any of these people. The decisions that they make will have absolutely no effect on you nor will any of your decisions affect them. Most importantly, your decisions will be strictly anonymous and cannot be linked to you in any way.

The experiment is conducted as follows:

• Everyone in Room A and in Room B has been allocated $5.

Earnings: [People in Room A have earned $10 for completing the 30‐minute task and people in Room B have earned $5 for completing the 30‐minute task.]

• The person in Room A (YOU) has been provisionally allocated an additional $5. Participants in Room B have not been allocated this additional $5.

• Your decision is a simple one: decide what portion, if any, of this $5 to transfer to the person you are paired with in Room B. Your choice can be anywhere from $0 to $5, in $0.50 increments. Your take‐home earnings from this experiment will be the summation of your initial $5 allocation and the money from this choice exercise. Likewise, the earnings of the person in Room B will be their initial $5 allocation added to the transfer amount from this exercise.

Take $1: [Your decision is a simple one: decide what portion, if any, of this $5 to transfer to the person you are paired with in Room B. You can also transfer a negative amount: i.e., you can take up to $1 from the person in Room B. Thus, your transfer choice can be anywhere from −$1 to $5, in $0.50 increments. Your take‐home earnings from this experiment will be the summation of your initial $5 allocation and the money from this choice exercise. Likewise, the earnings of the person in Room B will be their initial $5 allocation added to the outcome of this choice exercise.]

Take $5: [Your decision is a simple one: decide what portion, if any, of this $5 to transfer to the person you are paired with in Room B. You can also transfer a negative amount: i.e., you can take up to $5 from the person in Room B. Thus, your transfer choice can be anywhere from −$5 to $5, in $0.50 increments. Your take‐home earnings from this experiment will be the summation of your initial $5 allocation and the money from this choice exercise. Likewise, the earnings of the person in Room B will be their initial $5 allocation added to the outcome of this choice exercise.]

Take $5 Earnings: [Your decision is a simple one: decide what portion, if any, of $5 to transfer to the person you are paired with in Room B. You can also transfer a negative amount: i.e., you can take up to $5 from person B's earnings. Thus, your transfer choice can be anywhere from −$5 to $5, in $0.50 increments. Your take‐home earnings from this experiment will be the summation of your initial $5 allocation and the money from this choice exercise. Likewise, the earnings of the person in Room B will be their initial $5 allocation added to the outcome of this choice exercise.]

You will have five minutes to come to a decision about your choice. Please do not talk to the other people in this room until your session is completed. Do not be concerned if other people make their decisions before you, we will not collect the forms until after 5 minutes.

Are there any questions? Please circle your transfer below:

$0 $0.50 $1 $1.50 $2 $2.50 $3 $3.50 $4 $4.50 $5

Take $1: [−$1 −$0.50] added

Take $5 and Earnings: [−$5? −$0.50] added.